### PR TITLE
chore: election benchmarking skeleton improvement

### PR DESCRIPTION
**Pull request by syan095**
# Pull Request

## Checklist

Please conduct a thorough self-review before opening the PR.

- [x] I am confident that the code works.
- [ ] I have written sufficient tests.
- [ ] I have written and tested required migrations.
- [ ] I have updated documentation where appropriate.

## Summary
Benchmark/election benchmark skeleton improv

### CODE REVIEW

**Comment by syan095** [comment]
Feel free to merge if you guys think this is OK

File: state-chain/pallets/cf-elections/src/lib.rs [hunk 1]
```diff
@@ -251,6 +253,11 @@ pub mod pallet {
 				Self {}
 			}
 		}
+		impl Default for CorruptStorageError {
```
Comment (kylezs) [inline state-chain/pallets/cf-elections/src/lib.rs:256]: why do we need this?

**Review by Janislav — APPROVED** [review]

File: state-chain/pallets/cf-elections/src/lib.rs (hunk 1 above)
Comment (kylezs) [reply state-chain/pallets/cf-elections/src/lib.rs:256]: guessing it was clippy suggesting this... it's also unused anyway

File: state-chain/pallets/cf-elections/src/vote_storage/composite.rs [hunk 2]
```diff
@@ -232,3 +232,41 @@ generate_vote_storage_tuple_impls!(tuple_1_impls: (A));
...
+/// Implementation of BenchmarkValue for the concrete types used in our Runtime.
+/// Needed for Solana Witnessing.
+use cf_chains::{
+	benchmarking_value::BenchmarkValue,
+	sol::{SolAddress, SolHash},
+	Solana,
+};
+use frame_support::BoundedBTreeMap;
+use sp_core::ConstU32;
+use tuple_4_impls::{CompositeSharedData, CompositeVote};
```
Comment (dandanlen) [inline state-chain/pallets/cf-elections/src/vote_storage/composite.rs:248]: Can we put all of this inside a submodule? (can be in the same file)

I'm just not sure what the consequences are of pulling all these imports into the macro scope.

File: state-chain/pallets/cf-elections/src/vote_storage/composite.rs [hunk 3]
```diff
@@ -232,3 +232,41 @@ generate_vote_storage_tuple_impls!(tuple_1_impls: (A));
...
+			ConstU32<MAXIMUM_CHANNELS_PER_ELECTION>,
+		>,
+		SolHash,
+	>
+{
+	#[cfg(feature = "runtime-benchmarks")]
+	fn benchmark_value() -> Self {
+		CompositeVote::A(1_000u64)
+	}
+}
```
Comment (dandanlen) [inline state-chain/pallets/cf-elections/src/vote_storage/composite.rs:265]: Can we not implement this generically? 

```suggestion
impl<A, B, C, D> BenchmarkValue for CompositeVote<A, B, C, D>
	where A: BenchmarkValue,
{
	#[cfg(feature = "runtime-benchmarks")]
	fn benchmark_value() -> Self {
		CompositeVote::A(A::benchmark_value())
	}
}
```

And the same for SharedData ?

**Pull request merged**